### PR TITLE
ertyurk-patch-cache-folder-warning-section-3-lesson-1

### PR DESCRIPTION
**Pull request by ertyurk**
Warning message added to whom may choose to not create assets folder under the project

### CODE REVIEW

File: Solana_NFTs/en/Section_3/Lesson_1_Call_Candy_Machine_From_App.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ REACT_APP_SOLANA_RPC_HOST=
 
 Let's go one by one (note: **no** quotation symbols needed here).
 
+**Warning** `.cache/devent-temp` can be found on the top of `assets` folder after running Metaplex command in previous steps.
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_3/Lesson_1_Call_Candy_Machine_From_App.md:37]: Thanks for this!! 

This is definitely the right approach here, but I think we can change up the wording here to look a little something like this:

**Note**: `.cache/devent-temp` can be found at the root of your folder after running the Metaplex command in previous steps.

**Review by adilanchian — CHANGES_REQUESTED** [review]
Lemme know your thoughts!

**Comment by adilanchian** [comment]
Looks like there is a merge conflict here as well!

File: Solana_NFTs/en/Section_3/Lesson_1_Call_Candy_Machine_From_App.md (hunk 1 above)
Comment (ertyurk) [reply Solana_NFTs/en/Section_3/Lesson_1_Call_Candy_Machine_From_App.md:37]: Hey @adilanchian , Let me add that, thank you 👍🏼

**Comment by ertyurk** [comment]
@adilanchian I resolved the conflict, sorry for the extra spacing :)

**Comment by adilanchian** [comment]
Hey @ertyurk thanks so much for this! Look good to me :)

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION 

😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎😎

**Comment by adilanchian** [comment]
@all-contributors please add @ertyurk  for doc

**Pull request merged**